web/src/routes/(user)/search/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -349,13 +350,7 @@
     {/if}
     <section id="search-content" class="relative bg-immich-bg dark:bg-immich-dark-bg">
       {#if searchResultAssets.length > 0}
-        <GalleryViewer
-          assets={searchResultAssets}
-          {assetInteraction}
-          onIntersected={loadNextPage}
```

Comment:
Note to self, can't remove onIntersected as that is what loads the subsequent page of results when scrolling.